client/app/bundles/course/assessment/submission/containers/VoiceResponseAnswer.jsx

```diff
@@ -0,0 +1,209 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Field } from 'redux-form';
+import { injectIntl, intlShape, defineMessages, FormattedMessage } from 'react-intl';
+import FlatButton from 'material-ui/FlatButton';
+import MicIcon from 'material-ui/svg-icons/av/mic';
+import StopIcon from 'material-ui/svg-icons/av/stop';
+import SingleFileInput from 'lib/components/redux-form/SingleFileInput';
+import recorderHelper from '../../utils/recorderHelper';
+import sharedConstants from '../../../../../lib/constants/sharedConstants';
+import {
+  setRecording,
+  setNotRecording,
+  recorderComponentMount,
+  recorderComponentUnmount,
+} from '../actions/index';
+
+const translations = defineMessages({
+  startRecord: {
+    id: 'course.assessment.submission.answer.startRecord',
+    defaultMessage: 'Start Record',
+  },
+  stopRecord: {
+    id: 'course.assessment.submission.answer.stopRecord',
+    defaultMessage: 'Stop Record',
+  },
+  chooseVoiceFileExplain: {
+    id: 'course.assessment.submission.answer.chooseVoiceFileExplain',
+    defaultMessage: 'Drag your voice file here, or click to select voice file. \
+                     Only wav and mp3 files are supported. Alternatively, you may press the\
+                     record button below to record through your microphone',
```

Comment:
`Drag your audio file here, or click to select audio file. Only wav and mp3 format is supported. Alternatively, you may used the recorder below to record your response.`

Reply:
I think it should be ```Only wav and mp3 formats are...```. I will change it.